src/Integration.UnitTests/Connection/ConnectControllerTests.cs

```diff
@@ -378,6 +378,19 @@ public void ConnectionController_DontWarnAgainCommand_Status()
             testSubject.DontWarnAgainCommand.CanExecute().Should().BeTrue();
         }
 
+        [TestMethod]
+        public void ConnectionController_OrganizationsAreNotForget()
```

Comment:
I'm missing something - how does this test check that organizations are not forgotten?

Reply:
@duncanp-sonar sorry, that should not be here. Good catch!